### ISSUE TITLE
About the network inflation factors

### ISSUE THREAD

**Issue by hex0102** [itayhubara/BinaryNet.pytorch#7]
As I find out in the code, layers in VGG network, ResNet have an inflation factor.
Could someone please help clarify this? 
Why need to inflate the network? Is there a reference to address this question?
I also checked the tensorflow repo for the BNN network, there's no inflation factor.

**Comment by itayhubara** [comment]
you don't have to inflate the network but it improves accuracy. WRPN paper: https://arxiv.org/abs/1709.01134

**Comment by hex0102** [comment]
@itayhubara It seems like that for VGG_CIFAR10, the attained accuracy for the inflated network doesn't improve very much (limited to 1%). Are there any reasons to explain this?

**Comment by itayhubara** [comment]
VGG_CIFAR10 is a very large network to begin with...
